ambuda/views/proofing/main.py

```diff
@@ -242,35 +240,25 @@ def create_project_status(task_id):
 @bp.route("/recent-changes")
 def recent_changes():
     """Show recent changes across all projects."""
-    num_per_page = 100
+    per_page = 50
 
     # Exclude bot edits, which overwhelm all other edits on the site.
     bot_user = q.user(consts.BOT_USERNAME)
     assert bot_user, "Bot user not defined"
 
-    session = q.get_session()
-    recent_revisions = (
-        session.query(db.Revision)
-        .options(orm.defer(db.Revision.content))
-        .filter(db.Revision.author_id != bot_user.id)
-        .order_by(db.Revision.created.desc())
-        .limit(num_per_page)
-        .all()
-    )
-    recent_activity = [("revision", r.created, r) for r in recent_revisions]
-
-    recent_projects = (
-        session.query(db.Project)
-        .order_by(db.Project.created_at.desc())
-        .limit(num_per_page)
-        .all()
+    q.get_session()
+    recent_revisions = db.paginate(
+        db.select(Revision)
+        # Defer slow columns
+        .options(orm.defer(Revision.content))
+        # Avoid bot changes, which dominate the logs.
```

Comment:
Good choice to avoid bot changes. We may come back to have an alternative mechanism to view them. I agree that we should skip them here.